docs/execution_providers/Vitis-AI-ExecutionProvider.md

```diff
@@ -0,0 +1,66 @@
+<p align="center">
+  <img src="./images/Vitis-AI.png">
+</p>
+
+# Vitis-AI Execution Provider
+
```

Comment:
Would be good to have some contact information like github ids that should be assigned to any issues related to the EP.